modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -58,10 +63,18 @@ class Message extends Component {
     delete messageDataProps.component
 
     const props = {
-      ..._.pick(this.props, ['isLastGroup', 'isLastOfGroup', 'onSendData', 'onFileUpload', 'sentOn']),
+      ...pick(this.props, [
+        'isLastGroup',
+        'isLastOfGroup',
+        'isBotMessage',
+        'onSendData',
+        'onFileUpload',
+        'sentOn',
+        'store'
+      ]),
       ...messageDataProps,
       keyboard: Keyboard,
-      children: wrapped && <Message bp={this.props.bp} keyboard={Keyboard} noBubble={true} payload={wrapped} />
+      children: wrapped && <Message {...this.props} keyboard={Keyboard} noBubble={true} payload={wrapped} />
```

Comment:
This bugs me a little (I know it was previously written this way), you render a Message as children of a custom Message component i that right ?

Reply:
Somehow yes, I've encountered an issue when I wrapped all existing components for the dev tools. I'm not finished with it tho, I still need to test different scenarios